manifest.json

```diff
@@ -1,10 +1,6 @@
 {
   "name": "command-config",
   "description": "Edits UbiquityOS configuration files based on natural language queries.",
-  "ubiquity:listeners": [
-    "issue_comment.created",
```

Comment:
Aren't these required for the command to be captured? Otherwise how are we supposed to invoke it?